conanfile.py

```diff
@@ -0,0 +1,48 @@
+from conans import ConanFile, CMake
+from conans.tools import load
+import re
+
+
+def get_version():
+    try:
+        content = load("CMakeLists.txt")
+        version = re.search(r"^\s*project\(resource_pool\s+VERSION\s+([^\s]+)", content, re.M).group(1)
+        return version.strip()
+    except Exception:
+        return None
+
+
+class ResourcePool(ConanFile):
+    name = 'resource_pool'
+    version = get_version()
+    license = 'MIT'
+    url = 'https://github.com/elsid/resource_pool'
+    description = 'Conan package for elsid resource pool'
+
+    exports_sources = 'include/*', 'CMakeLists.txt', 'resource_poolConfig.cmake', 'LICENCE', 'AUTHORS'
+
+    generators = 'cmake_paths'
+    requires = 'boost/1.70.0@conan/stable'
+
+    # Internal option for ozo devs to use cmake for easier setup
+    options = {'ozo_require_gtest': [True, False]}
+    default_options = {'ozo_require_gtest': False}
```

Comment:
Why do you mention ozo here and why this option is not used anywhere else? I'm not familiar with conan. Is this option used to build ozo? But it's not mentioned in https://github.com/yandex/ozo/pull/143 . I don't quite understand what is logic behind it.

Reply:
There is no logic behind that, it's overly enthusiastic copy&paste. I originally didn't have a separate conanfile for `resource_pool` and just had the ozo one pack everything.
For this PR, I figured a separate package is better, as the library is very useful in standalone scenarios. I just copied it over, changed the names and symbols and removed everything ozo-specific -- or so I thought.

As this didn't impact the build process, I simply didn't notice it. Sorry for the inconvenience.